lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -243,8 +135,7 @@ def valid_environment?
         if ipa_client_configured?
           show_current_configuration
           return false unless agree("\nIPA Client already configured on this Appliance, Un-Configure first? (Y/N): ")
-          ipa_client_unconfigure
-          unconfigure_httpd
+          deactivate
```

Comment:
Unsure about this one.

`ExternalHttpdConfiguration` should not call into `ExternalHttpdAuthentication`.